frontend/src/services/roadmapService.ts

```diff
@@ -19,6 +19,9 @@ export default {
       emailData.bcc = parseCSV(emailData.bcc);
     }
 
-    return appAxios().put('roadmap', { activityId, emailData });
+    const attachmentIds = emailData.attachments;
+    delete emailData.attachments;
+
+    return appAxios().put('roadmap', { activityId, attachmentIds, emailData });
```

Comment:
Something seems off about this - generally would be better to explicitly define the newly constructed object key and values than to have it be implicitly put together for safety/typing purposes. There might be a way of assigning this, without requiring the use of a delete statement.